ccmbioinfo/lfs
# CARDS-2392: Switch YE clarity import to pull data from new table PatientActivity_data_for_PtExpSurveyApp

To test:

- build with `mvn clean install -Pdocker`
- in `compose-cluster/mssql` run python3 generate_test_YE_sql.py -n 100 prems_sample.sql`
- in `compose-cluster` run `python3 generate_compose_yaml.py --mssql --cards_project cards4prems --oak_filesystem --dev_docker_image --composum --adminer` to generate a docker-compose file
- start docker with `docker-compose build && docker-compose up`
- at `http://localhost:1435/?mssql=mssql&username=sa&db=master&ns=path&import=` import the generated `prems_sample.sql` file
- open `http://localhost:8080/Subjects.importClarity` to start importing the patients
- check that patients and visits have been imported
- check that patients with `EMAIL_CONSENT_YN=No , MYCHART_STATUS=Yes` have been imported and have consent set to Yes
- check that patients with `UHN_ICC_PATIENT_ELIGIBILITY=Enrolled ; UHN_ICC_STATUS = New Patient` have their visit marked as `in-progress`
- check that rows with `PRIMARY_DX_NAME` of `Agitation` or `Alcohol withdrawal` were not imported
